### ISSUE TITLE
Admin Read this Before Close It

### ISSUE THREAD

**Issue by animek04** [Lunat1q/Catchem-PoGo#994]
did you realize that there are any mirrors for this app? I mean, a lot of sites with unofficial download links.
There is a lot of users on youtube offering ths also for money.. You know that right?

@Lunat1q , here we are not trying to copy, steal, take credit, lie, scam, nothing. We just created a channel, we have now 280 members so farm and we're trying to hel everyone.

Is there another Discord Channel? Ok. There could be more than 10, it is normal, it is internet.

So, please, fell free to join our channel and lets talk live for a few minutes.. they will be more than happy to see the "master" itself....

When you come in, I'll make you Administrator at once!

https://discord.gg/6z5AMX5

**Comment by lonelies** [comment]
why are you so desperate bringing him there? got beer?

**Comment by wagas** [comment]
better let your 280 members go to the OFFICIAL DISCORD SERVER FOR THE BOT. =)

**Comment by animek04** [comment]
Nope, just trying to help but not allowed to... so it is frustrating...

**Comment by lonelies** [comment]
No means, No. That's it.

**Comment by animek04** [comment]
Trying means trying... That is it.

**Comment by lonelies** [comment]
You invited him, he rejected. Can't you respect that? lmao

**Comment by Lunat1q** [comment]
There is Official server with OFFICIAL HELP channel. For everyone. If u will continue to spam here, i'll close issues at all.

**Comment by kaktusisko** [comment]
Mayby leave one offtopic issue?

**Comment by Lunat1q** [comment]
there is discord for offtopic.

**Comment by wagas** [comment]
you are helping others but still your advertising your own page.. why not just help?
